### PR TITLE
Generic CI for libraries

### PR DESCRIPTION
**Pull request by nicolaicinali**

### CODE REVIEW

File: .github/workflows/library-ci.yml
Comment (segabriel) [inline .github/workflows/library-ci.yml:6]: So you added this option but no where use that.

**Review by segabriel — APPROVED** [review]
LGTM but wait for @artem-v approval please.

File: .github/workflows/library-ci.yml
Comment (nicolaicinali) [reply .github/workflows/library-ci.yml:6]: Yes, not sure which options to use, to be honest, but I've added just to make sure that we can use this in case we need.
Like we use for some of the services (Example template):
```
    with:
      mvn-verify-opts: |
        [
          "-pl :service-name -Denforcer.skip=true -Pcoverage",
          "-pl :service-name1,:service-name2 -Denforcer.skip=true -Pcoverage",
          "-pl :service-name -Denforcer.skip=true -Pcoverage"
        ]
```

**Review by artem-v — APPROVED** [review]

**Comment by artem-v** [comment]
> LGTM but wait for @artem-v approval please.

Im ok with changes too.

**Pull request merged**
